### ISSUE TITLE
"SSL peer certificate or SSH remote key was not OK"

### ISSUE THREAD

**Issue by Kopflala** [transmission-remote-gtk/transmission-remote-gtk#40]
Server-side, transmission-daemon is running through stunnel. Connection works fine from another client. Used to work fine from transmission-remote-gtk (1.1.1 from Ubuntu repository). Connection now fails with message "**SSL peer certificate or SSH remote key was not OK**".

Between now and then, server certificate changed. Error message is not helpful.

**Comment by ajf8** [comment]
I just tested the SSL support using stunnel4 on the latest debian, using a self signed cert I made. It worked fine. With master and the version in debian, and the "validate certificate" disabled as per default.

Sorry, I suspect there is some problem with your SSL cert or stunnel setup, but I have no idea what.

If I enable validation it gives me a fairly helpful error that the cert cannot be authenticated with any CA.
